sormas-e2e-tests/src/main/resources/configuration/properties/environment/test-performance.properties

```diff
@@ -16,4 +16,6 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
-ENVIRONMENT_URL=https://test-performance.sormas.netzlink.com/sormas-ui/login#!cases/?includeCasesFromOtherJurisdictions=false&dateFilterOption=DATE
\ No newline at end of file
+ENVIRONMENT_URL=https://test-performance.sormas.netzlink.com
+REST_USER=SurvOff
+REST_PASSWORD=SurvOff38118
```

Comment:
I would not expose any credentials to a public repo

Reply:
Good point, I ask for this and it is ok, because the repo as well is open source. This will be tackled when we will user the jenkins/GitHub actions